libr/asm/p/asm_snes.c

```diff
@@ -5,11 +5,33 @@
 #include <r_asm.h>
 #include <r_lib.h>
 #include "../arch/snes/snesdis.c"
+#include "asm_snes.h"
+
+struct snes_asm_flags* snesflags = NULL;
+
+bool snes_asm_init (void* user) {
+	if (!snesflags) snesflags = malloc(sizeof( struct snes_asm_flags ));
```

Comment:
use r2-indent (braces on all if blocks, even the oneliner ones). add spaces *before* the (, not after